web/src/findLogLine.ts

```diff
@@ -0,0 +1,22 @@
+const LINE_ID_ATTR_NAME = "data-lineid"
+
+export default function findLogLineID(
+  el: HTMLElement | Node | null
+): string | null {
+  if (el === null) {
+    return null
+  }
+
+  if (
+    typeof (el as HTMLElement).getAttribute === "function" &&
```

Comment:
I think there's probably a better way to do this type juggling but I'm not sure what it is.

Reply:
function findLogLineID(node: Node | null)

let el = (node as HTMLElement)
maybe?

it's a little bit weird that you have a variable called `el` that's not an element